maths/binomial_distribution.py

```diff
@@ -22,22 +17,20 @@ def binomial_distribution(k, n, p) -> float:
     >>> binomial_distribution (2, 4, 0.5)
     0.375
 
-    >>> binomial_distribution (2, 4, -0.5)
-    Traceback (most recent call last):
-    ...
-    raise ValueError("p - Probability has to be in range of 1 - 0")
-    ValueError: p - Probability has to be in range of 1 - 0
     """
-    if k > n:
-        raise ValueError("""k must be lower or equal to n""")
-    if n < 0 or k < 0 or type(k) != int or type(n) != int:
-        raise ValueError("the function is defined for non-negative integers k and n")
-    if p > 1 or p < 0:
-        raise ValueError("p - Probability has to be in range of 1 - 0")
-    probability = (p**k)*(1-p)**(n-k)
+    if successes > trials:
+        raise ValueError("""successes must be lower or equal to trials""")
+    if trials < 0 or successes < 0:
+        raise ValueError("the function is defined for non-negative integers")
+    if type(successes) != int or type(trials) != int:

```

Comment:
```suggestion
    if not isinstance(successes, int) or not isinstance(trials, int):
```
As discussed in PEP8